integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -49,8 +48,9 @@ const INCREASED_EPOCH_LENGTH: u64 = 8;
 /// Garbage collection window length.
 const GC_NUM_EPOCHS_TO_KEEP: u64 = 3;
 
-/// Maximum number of epochs under which the test should finish.
-const TESTLOOP_NUM_EPOCHS_TO_WAIT: u64 = 8;
+/// Default number of epochs for resharding testloop to run.
+// TODO(resharding) Fix nearcore and set it to 10.
```

Comment:
Do you have any insight on why it fails with 10? If you do it might be good to capture that in a comment or issue.

If you don't have any relevant clue, feel free to ignore this comment

Reply:
Yes, this is the issue described here: https://near.zulipchat.com/#narrow/channel/407288-core.2Fresharding/topic/forknet/near/494174933